cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -155,6 +155,7 @@ import WACIError from '@/components/WACI/WACIError.vue';
 import WACILoading from '@/components/WACI/WACILoading.vue';
 import CredentialDetailsTable from '@/components/WACI/CredentialDetailsTable.vue';
 import OIDCShareOverview from '@/pages/OIDCShareOverview.vue';
+import JSONWebToken from '../classes/JSONWebToken';
```

Comment:
We have alias defined in our `vue.config.js`, so you can use the `@` identifier:
```suggestion
import JSONWebToken from '@/classes/JSONWebToken';
```